_docs/overview/key-concepts.md

```diff
@@ -34,6 +34,12 @@ This is analogous to storing coins in a padlocked box, with multiple keys and
 padlocks which open in different combinations. See the [Multisignature Security](/docs/overview/multi-signature-security)
 section for more detail.
 
+## Redeem script
+
+Multisignature requires each of the multiple private keys to be accompanied by a
+redeem script. This script, each of which is identical, defines how the keys
+combine to release funds. Losing all redeem scripts leads to the total loss of funds.
```

Comment:
Losing all redeem scripts would not lead to loss of funds **iff** a previous withdrawal was performed (such as the test withdrawal that the protocol recommends). Assuming one still knows the cold storage address, the withdrawal script can be found in the blockchain.

Reply:
Information added in new commit